.github/workflows/testing.yml

```
@@ -30,19 +30,39 @@ jobs:
             python -m pip install --upgrade pip
             pip install -r requirements.txt
 
+        - name: Check for tests
+          id: check_tests
+          run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:29: Double quote to prevent globbing and word splitting

(shellcheck)